src/chunk.js

```diff
@@ -452,6 +455,9 @@ class Chunks {
  */
 export function onIdle(win, minimumTimeRemaining, timeout, fn) {
   const startTime = Date.now();
+  /**
+   * @param {?IdleDeadline} info
```

Comment:
`{!IdleDeadline}` seems better since info is accessed without null check.

Reply:
done